.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+---
+name: tests
+
+on:
+  push:
+    branches: [master, develop]
```

Comment:
@ivangergo3 since I've seen that @novykh has commented on the other PRs that you have opened that we only need it for `master` maybe this should be changed as well! 😄

Reply:
@burbuli8ra this one is fine. The others do not have a `develop` branch.